run-tests

```diff
@@ -74,7 +74,7 @@ run_database_testsuite () {
           t=${line[1]}
           expect_broken=1
         fi
-        cmnd="./links --config=$1/config $1/$t.links"
+        cmnd="./links --config=$1/config --database=/plugins/pg_db/pg_database.cmxs $1/$t.links"
```

Comment:
This doesn't look right - it seems to me like it will look for the absolute path /plugins/pg_db/pg_database.cmxs rather than relative to the current path.  removing the initial "/" may fix that.